### PR TITLE
Fixed examples

### PR DESCRIPTION
**Pull request by joeyouss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +40,8 @@ This may be a timeout error. Add `timeout_ready=240000` to the Executor in your
 ```python
 flow = Flow().add(uses="foo", timeout_ready=240000)
 ```
+### Any error 
+This may be due to using an older version of hub. Please delete your cache using `rm -rf ~/.jina` and then try running again.
```
Comment (alexcg1) [inline README.md]: - `hub` -> `Jina Hub`

File: basics/2_executor_options/app.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 flow = (
     Flow()
     .add(
-        uses="jinahub+docker://SpacyTextEncoder",
+        uses="jinahub://SpacyTextEncoder",
         # Change to "medium" model for better encoding
         uses_with={"model_name": "en_core_web_md"},
         name="encoder",
```
Comment (alexcg1) [inline basics/2_executor_options/app.py:15]: you'll also need to add `install_requirements=True`

Comment (alexcg1) [reply basics/2_executor_options/app.py:15]: Do this for each occurrence

**Review by alexcg1 — CHANGES_REQUESTED** [review]

**Comment by joeyouss** [comment]
@alexcg1 changes made!

File: README.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ This may be a timeout error. Add `timeout_ready=240000` to the Executor in your
 flow = Flow().add(uses="foo", timeout_ready=240000)
 ```
 ### Any error 
-This may be due to using an older version of hub. Please delete your cache using `rm -rf ~/.jina` and then try running again.
+This may be due to using an older version of Jina Hub. Please delete your cache using `rm -rf ~/.jina` and then try running again.
```
Comment (alexcg1) [inline README.md]: Actually I messed that up:
- `an older version of Jina Hub` -> `older versions of Hub Executors`

File: basics/3_document_generators/app.py [hunk 4]
```diff
@@ -16,12 +16,14 @@
...
         uses_with={"model_name": "en_core_web_md"},
         name="encoder",
+        install_requirements=True
     )
     .add(
         uses="jinahub+docker://SimpleIndexer",
         uses_metas={"workspace": "workspace"},
         volumes="./workspace:/workspace/workspace",
         name="indexer",
+        install_requirements=True
```
Comment (alexcg1) [inline basics/3_document_generators/app.py]: Sorry, if we have `+docker` we don't need `install_requirements`

Comment (alexcg1) [reply basics/3_document_generators/app.py]: So keep it for spacy, remove it for simpleindexer

**Review by alexcg1 — COMMENTED** [review]
A few more changes. I shoulda caught them first time

**Comment by joeyouss** [comment]
@alexcg1 please check

**Review by alexcg1 — APPROVED** [review]
LGTM :+1: You can go ahead and merge this!

**Pull request merged**